pkgs/development/python-modules/marshmallow/default.nix

```diff
@@ -10,7 +10,7 @@
 
 buildPythonPackage rec {
   pname = "marshmallow";
-  version = "3.16.0";
```

Comment:
Is it okay to just update this? It might break other packages...

Reply:
As you can see in my nixpkgs-review comments, it indeed breaks a bunch of packages on x86_64-darwin, so I dropped the commit to update marshmallow and just added the dependency itself in a compatible version.